libtrellis/CMakeLists.txt

```diff
@@ -108,24 +108,28 @@ endif()
 
 find_package(Boost REQUIRED COMPONENTS program_options)
 
-get_property(LIB64 GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS)
-if ("${LIB64}" STREQUAL "TRUE")
-    set(LIBDIR "lib64")
+if (NOT APPLE)
+    get_property(LIB64 GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS)
+    if ("${LIB64}" STREQUAL "TRUE")
+        set(LIBDIR "lib64")
+    else()
+        set(LIBDIR "lib")
+    endif()
```

Comment:
IMHO, it might be more compact (and legible) here to simply do:
```
get_property(LIB64 ...
if (NOT APPLE AND "${LIB64}" STREQUAL "TRUE")
  ...
```